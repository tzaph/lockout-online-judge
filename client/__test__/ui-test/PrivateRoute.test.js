import React from "react";
import renderer from "react-test-renderer";
import PrivateRoute from "../../src/components/PrivateRoute";
import { BrowserRouter as Router } from 'react-router-dom';

describe("<PrivateRoute />", () => {
  it("Renders correctly", () => {
    const tree = renderer.create(<Router><PrivateRoute /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
