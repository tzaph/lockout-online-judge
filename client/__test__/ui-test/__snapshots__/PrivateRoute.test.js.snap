// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivateRoute /> Renders correctly 1`] = `null`;
